File: Lab_8/src/main/java/Accommodation.java
public class Accommodation {
    private int numberOfRooms;
    private double price;
    private double area;
    private Integer id = null;

    public Accommodation() {
    }

    public Accommodation(Integer id, int numberOfRooms, double price, double area) {
        this.numberOfRooms = numberOfRooms;
        this.price = price;
        this.area = area;
        this.id = id;
    }

    @Override
    public String toString() {
        return "\n\nAccommodation:" +
                "\nnumberOfRooms = " + numberOfRooms +
                "\nprice = " + price +
                "\narea = " + area;
    }

    public int getNumberOfRooms() {
        return numberOfRooms;
    }

    public void setNumberOfRooms(int numberOfRooms) {
        this.numberOfRooms = numberOfRooms;
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }

    public double getArea() {
        return area;
    }

    public void setArea(double area) {
        this.area = area;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }
}

File: Lab 14 (lab 5 python)/accommodation.py
from app import db


class Accommodation(db.Model):
    __tablename__ = "accommodations"
    id = db.Column('id', db.Integer, primary_key=True)
    number_of_rooms = db.Column('number_of_rooms', db.String)
    price = db.Column('price', db.Integer)
    area = db.Column('area', db.Integer)

    def __str__(self):
        return str("id = " + str(self.id) + "\nnumber of rooms = " +
                   str(self.number_of_rooms) + "\nprice = " + str(self.price) + "\narea = " + str(self.area))
